flixel/graphics/tile/FlxDrawTrianglesItem.hx

```diff
@@ -157,10 +168,30 @@ class FlxDrawTrianglesItem extends FlxDrawBaseItem<FlxDrawTrianglesItem>
 		var i:Int = 0;
 		var currentVertexPosition:Int = prevVerticesLength;
 
+		var _cos:Float = 1.0;
+		var _sin:Float = 0.0;
+		if (angle != 0)
+		{
+			_cos = FlxMath.fastCos(angle * FlxAngle.TO_RAD);
+			_sin = FlxMath.fastSin(angle * FlxAngle.TO_RAD);
```

Comment:
would you be willing to compare this with using `Math.sin` and cos to see if the lack of precision actually a noticeable difference, be sure to try angles > 360, too please

Reply:
I fixed the rest, but I'd still like you to test the precision of FastSin and cos